### PR TITLE
do not depend on the routing bundle for content basepath

### PR DESCRIPTION
**Pull request by dbu**
https://github.com/symfony-cmf/routing-bundle/pull/387 reveiled that we had an implicit dependency on the routing bundle for the phpcr-sitemap. this PR moves the setting to an seo bundle config

### CODE REVIEW

File: src/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -111,6 +111,7 @@ private function addPersistenceSection(NodeBuilder $treeBuilder)
                         ->canBeEnabled()
                         ->children()
                             ->scalarNode('manager_name')->defaultNull()->end()
+                            ->scalarNode('content_basepath')->defaultValue('/cms/content')->end()
```
Comment (dbu) [inline src/DependencyInjection/Configuration.php]: should we prepend this from the core bundle?

Comment (wouterj) [inline src/DependencyInjection/Configuration.php]: 👍

Comment (dbu) [inline src/DependencyInjection/Configuration.php]: but if we prepend, the default value here is null, right?

**Comment by dbu** [comment]
https://github.com/symfony-cmf/core-bundle/pull/227

File: src/DependencyInjection/Configuration.php (hunk 1 above)
Comment (wouterj) [inline src/DependencyInjection/Configuration.php]: No, we can leave the default here as is. If CoreBundle is registered, the default will be overriden by the value that is prepended. If CoreBundle isn't available, the default value will be used.

Comment (dbu) [inline src/DependencyInjection/Configuration.php]: cool, then lets merge, ok?

Comment (wouterj) [inline src/DependencyInjection/Configuration.php]: Jep

**Pull request merged**